api/routes/apds/get.endpoint.js

```diff
@@ -0,0 +1,40 @@
+const {
+  getDB,
+  getFullPath,
+  login,
+  request,
+  unauthenticatedTest,
+  unauthorizedTest
+} = require('../../utils.endpoint');
+
+describe('APD endpoint | GET /apds', async () => {
+  const db = getDB();
+  beforeAll(() => db.seed.run());
+  afterAll(() => db.destroy());
```

Comment:
One big downside to switching to Jest is having to add this to every set of tests that modify the database.  `tap` has an event that fires when tests are finished, and you can tap into that to destroy the database connection - if you don't destroy the connection, the test process hangs and eventually the test is failed due to timeout.  Jest doesn't have that, so we have to hook the `afterAll` teardown method.